app/Bus/Handlers/Commands/Metric/AddMetricPointCommandHandler.php

```diff
@@ -65,23 +72,23 @@ public function handle(AddMetricPointCommand $command)
 
     protected function findOrCreatePoint(AddMetricPointCommand $command)
     {
-        $buffer = Carbon::now()->subMinutes(5);
+        $buffer = Carbon::now()->subMinutes($this->config->get('setting.metric_threshold', 5));
```

Comment:
@jbrooksuk Agreed, per metric should be better. @GrahamCampbell We are removing the config :stuck_out_tongue: